### ISSUE TITLE
[Request] A cheat or separate plugin to change the max reserve ammo for any weapon you have.

### ISSUE THREAD

**Issue by danylopez123** [wootguy/Cheats#2]
I've been trying to search for a plugin that allows to change the max reserve ammo you can have but i had no luck.
I know there is this trigger "trigger_changemaxammo" that you can add for a map but that's just a trigger.
I might say that it would be better as a separate plugin because so anyone who joins can have the max reserve ammo already changed.

**Comment by wootguy** [comment]
yeah better as its own plugin, but I probably won't be the one to make that.
